t/info.t

```diff
@@ -160,4 +160,26 @@ is_json_GET(
   );
 }
 
+#/info/variation/:species
+{
+  my $variation_json = json_GET('/info/variation/homo_sapiens', 'Get analysis hash');
+  cmp_ok(scalar(keys %{$variation_json}), '==', 6, 'Ensuring we have the right number of sources available');
```

Comment:
I think this should be cmp_ok(scalar(@{$variation_json})
Message from travis: Not a HASH reference at t/info.t line 166.